src/glum/_glm.py

```diff
@@ -1279,7 +1279,8 @@ def _find_alpha_index(self, alpha):
         if not self.alpha_search:
             raise ValueError
         # `np.isclose` because comparing floats is difficult
-        isclose = np.isclose(self._alphas, alpha)
+        atol = 100.0 * np.finfo(alpha.dtype).eps
```

Comment:
`alpha` is a scalar (e.g. float or np.float32), so it doesn't have a `dtype` attribute.
```suggestion
        atol = 100.0 * np.finfo(type(alpha)).eps
```